src/py/reactpy/reactpy/backend/__init__.py

```diff
@@ -0,0 +1,22 @@
+import mimetypes
+from logging import getLogger
+
+_logger = getLogger(__name__)
+
+# Fix for missing mime types due to corrupt Windows registry
+if not mimetypes.inited:
+    mimetypes.init()
+MIME_TYPES = {
+    ".js": "application/javascript",
+    ".css": "text/css",
+    ".json": "application/json",
+}
+for extension, mime_type in MIME_TYPES.items():
+    if not mimetypes.types_map.get(extension):  # pragma: no cover
+        _logger.warning(
+            "Mime type '%s = %s' is missing. Please research how to "
+            "fix missing mime types on your operating system.",
+            extension,
+            mime_type,
+        )
+        mimetypes.add_type(mime_type, extension)
```

Comment:
Given that this is a Windows-only issue can we put this in an `if os.name == "nt"` block?

Reply:
It's technically not just a Windows issue, but since the Windows registry is a free-for-all hellhole it's far more likely to happen on Windows.